scalafix-core/shared/src/main/scala/scalafix/internal/util/DenotationOps.scala

```diff
@@ -0,0 +1,32 @@
+package scalafix.internal.util
+
+import scala.meta._
+
+object DenotationOps {
+  val defaultDialect =
+    dialects.Scala212.copy(allowMethodTypes = true, allowTypeLambdas = true)
+
+  def resultType(
```

Comment:
the name is up to debate. `resultType` is general enough, but it's a bit of a stretch when referring to simple `val`/`var`

Reply:
Result type is the correct nomenclature from scalac 👍